frontend/src/Components/CreatePostForm.js

```diff
@@ -0,0 +1,129 @@
+import { useState } from "react";
+import { 
+  Button, 
+  TextField, 
+  Card, 
+  CardContent, 
+  Typography, 
+  Box, 
+  CircularProgress,
+  IconButton
+} from "@mui/material";
+import EmojiEmotionsIcon from "@mui/icons-material/EmojiEmotions";
+// Fix the import paths to use relative paths instead of alias
```

Comment:
this comment should be removed. it is not alias, it is package name. Best practice is to use aliases though.